packages/react/src/slider/thumb/useSliderThumb.ts

```diff
@@ -110,17 +109,16 @@ export function useSliderThumb(parameters: useSliderThumb.Parameters): useSlider
     return {
       position: 'absolute',
       [{
-        horizontal: 'left',
-        'horizontal-reverse': 'right',
+        horizontal: 'insetInlineStart',
```

Comment:
The CSS logical property counterparts of positioning properties like `left`/`top` are supported by all our supported browsers
https://caniuse.com/mdn-css_properties_inset-inline-start